src/avfoundation.cs

```diff
@@ -13513,7 +13513,7 @@ interface AVCompositionTrackFormatDescriptionReplacement : NSSecureCoding {
 		CMFormatDescription ReplacementFormatDescription { get; }
 	}
 
-	delegate /* OSStatus */ int AVAudioSourceNodeRenderHandler (bool isSilence, AudioTimeStamp timestamp, uint frameCount, ref AudioBuffers outputData);
+	delegate /* OSStatus */ int AVAudioSourceNodeRenderHandler (ref bool isSilence, ref AudioTimeStamp timestamp, uint frameCount, ref AudioBuffers outputData);
```

Comment:
Another question - do we have unit tests for delegates + should I add one for this?

Reply:
Please add a test, because they're all different so there's no generic way to test all delegates.